BaseApi/V1/Infrastructure/Constants.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace BaseApi.V1.Infrastructure
+{
+    public static class Constants 
```

Comment:
I think this should not be in infrastructure for the same reason for the middleware 's comment

Reply:
Where should I move this one? I was thinking that constants could hold constants from all over the place.